packages/runtime/container-runtime/src/containerRuntime.ts

```diff
@@ -2150,13 +2187,23 @@ export class ContainerRuntime extends TypedEventEmitter<IContainerRuntimeEvents>
      */
     public submitSignal(type: string, content: any) {
         this.verifyNotClosed();
-        const envelope: ISignalEnvelope = { address: undefined, contents: { type, content } };
+        const envelope: ISignalEnvelope = {
+            address: undefined,
+            clientSignalSequenceNumber: ++this._perfSignalData.signalSequenceNumber,
```

Comment:
would this sequence number be stable across different runtimes or versions of the runtime? Or are we assuming we're going to be missing quite a few messages?

Reply:
That is actually a great question. I will follow-up with Vlad on our 1-1 as I'm assuming we would be good as long as the message does not expire.